hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/mapreduce/MapReduceBackupCopyJob.java

```diff
@@ -425,4 +435,25 @@ public void cancel(String jobId) throws IOException {
     }
   }
 
+  protected static List<String> parseDistCpOptions(Configuration conf) {
+    List<String> extraArgsFromConf = new ArrayList<>();
+
+    for (DistCpOptionSwitch optionSwitch : DistCpOptionSwitch.values()) {
+      String configLabel = optionSwitch.getConfigLabel();
+      if (conf.get(configLabel) != null) {
```

Comment:
Do we want to add some prefix for these distcp configs?

Like [hbase.backup.copy.]distcp.xxxx

Reply:
Nice idea